dist/api_data.py

```diff
@@ -451,15 +449,28 @@ def __ge__(self, other):
 ]
 
 lsm_meta = file_config + lsm_config + [
-    Config('last', '', r'''
+    Config('last', '0', r'''
```

Comment:
@ddanderson I had to initialize `last` for tiered work and did the same for LSM, per our recent discussion about LSM chunk numbers.